apps/wpcom-block-editor/src/wpcom/features/live-preview/hooks/use-can-preview-but-need-upgrade.ts

```diff
@@ -139,10 +138,7 @@ export const useCanPreviewButNeedUpgrade = (
 
 	const handleCanPreviewButNeedUpgrade = useCallback(
 		( previewingTheme: ReturnType< typeof usePreviewingTheme > ) => {
-			const livePreviewUpgradeTypes = [ WOOCOMMERCE_THEME, PREMIUM_THEME ];
-			if ( config.isEnabled( 'themes/tiers' ) ) {
-				livePreviewUpgradeTypes.push( PERSONAL_THEME );
-			}
+			const livePreviewUpgradeTypes = [ WOOCOMMERCE_THEME, PREMIUM_THEME, PERSONAL_THEME ];
 
 			/**
 			 * Currently, Live Preview only supports upgrades for WooCommerce and Premium themes.
```

Comment:
trivial: this comment is now incorrect

Reply:
That's what I hate about comments, you need to maintain the code + the comments. I'll update it.

Reply:
Well, with the feature check that won't work added back, this is now true again!!